### PR TITLE
Specify Heap size, update .gitignore to ignore deploy.sh script

### LINKED ISSUE #191: Deployment should specify heap size
The following exceptions are thrown Due to heap size exceeded. I am afraid, we would need to create a custom entrypoint to the geenrated image or append startup argument, which will be appended after the entrypoint as a cmd argument. (This way podman/docker works).

This very question for quarkus on [StackOverflow](https://stackoverflow.com/questions/69899213/how-to-defiine-quarkus-heap-space-size) which refers to docs of [graalvm](https://www.graalvm.org/latest/reference-manual/native-image/optimizations-and-performance/MemoryManagement/#java-heap-size)

See [this JVM option](https://quarkus.io/guides/container-image#quarkus-container-image-openshift_quarkus-openshift-jvm-arguments) or the one below for native application. I am not sure, as it seems this flag is treated as classic JVM option eventhough it's native.

```java
2024-03-20 20:27:48,675 ERROR [io.qua.githubapp] (executor-thread-1061) Error handling delivery ad1a32f0-e6f7-11ee-841a-3f759b6fe848
› Repository: wildfly/wildfly
› Event: pull_request.edited
› Context: https://github.com/wildfly/wildfly/pull/17741
Exception: java.lang.OutOfMemoryError: Garbage-collected heap size exceeded. Consider increasing the maximum Java heap size, for example with '-Xmx'.
```

I do not want to play with it just yet, I don't know the resource quotas on the cluster. Please, have a look @xstefank

### PR DESCRIPTION
**Pull request by The-Huginn**
Resolves #191

### CODE REVIEW

**Comment by The-Huginn** [comment]
Also, updates quarkus version

File: .github/wildfly-bot.yml
Comment (xstefank) [inline .github/wildfly-bot.yml:1]: what is this?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 Contributing to WildFly Github Bot
 ==================================
 
-Welcome to the WildFly Github Bot project and thank you for deciding to contribute to this project! We welcome contributions from the community. This guide will walk you through the steps for getting started on our project.
+Welcome to the WildFly Github Bot project and thank you for deciding to contribute to this project! We welcome asdcontributions from the community. This guide will walk you through the steps for getting started on our projectasd.
```
Comment (xstefank) [inline CONTRIBUTING.md:4]: ?

**Review by xstefank — COMMENTED** [review]
@The-Huginn something went wrong in this PR, please update it

**Comment by The-Huginn** [comment]
Sorry, that was left from testing my other app, should be ready for review now

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -2,9 +2,11 @@ quarkus.openshift.route.expose=true
 quarkus.openshift.labels."app"=wildfly-bot
 quarkus.openshift.env.secrets=wildfly-bot
 quarkus.openshift.resources.requests.cpu=100m
-quarkus.openshift.resources.requests.memory=200Mi
+quarkus.openshift.resources.requests.memory=192Mi
```
Comment (xstefank) [inline src/main/resources/application.properties:5]: do we need 192Mi when we max heap on 128M? Not sure about the numbers.

Comment (The-Huginn) [reply src/main/resources/application.properties:5]: those should be 2 different numbers. Where 128Mi is just for the heap itself, which JVM operates upon. 192Mi is for pod as a whole

**Review by xstefank — APPROVED** [review]

**Pull request merged**